modules/rosa-configure.adoc

```diff
@@ -195,12 +195,37 @@ Verify that the AWS quotas are configured correctly in a specific region:
 $ rosa verify quota --region=us-west-2
 ----
 
+[id="rosa-download-rosa-client_{context}"]
+== download rosa
+
+Download the latest compatible version of the Red Hat OpenShift Service on AWS CLI (`rosa`).
```

Comment:
Same comment about using {product-title}

Reply:
As above, this section wouldn't make sense with any other product name

Reply:
That might be true, but that's not the only reason we use attributes.  Even for core OCP, we never hard code "OpenShift Container Platform", even if that's the only way it's used - it's also to help make sure no typos are being introduced. So it's just still a best practice to use the attribute, and our guidelines are here that mention to use it when possible: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-title-and-version

Reply:
Good point, thankyou!